app/src/main/res/values/strings_opt_in_notification.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="opt_in_notification_title">Get Tips and Encouragement</string>
+    <string name="opt_in_notification_body">Stay equipped for conversations.\nAllow notifications today.</string>
+    <string name="opt_in_notification_allow_notifications">Allow Notifications</string>
+    <string name="opt_in_notification_notification_settings">Notification Settings</string>
+    <string name="opt_in_notification_maybe_later">Maybe Later</string>
```

Comment:
let's just add this strings to the `strings_dashboard.xml` file under a new "section"